objectRecognitionSource/src/test/java/io/vantiq/extsrc/objectRecognition/neuralNet/TestYoloProcessor.java

```diff
@@ -948,18 +978,32 @@ public void testQuery() throws ImageProcessingException, InterruptedException {
                     + "             \"type\": \"yolo\""
                     + "         }"
                     + "}";
+    
+    String neuralNetJSON5 =
+            "{"
+                    + "     \"neuralNet\":"
+                    + "         {"
+                    + "             \"pbFile\": \"" + PB_FILE_608 + "\", "
+                    + "             \"metaFile\": \"" + META_FILE_608 + "\", "
+                    + "             \"type\": \"yolo\""
+                    + "         }"
+                    + "}";
 
-    List<Map> getExpectedResults() throws JsonParseException, JsonMappingException, IOException {
+    List<Map> getExpectedResults(boolean useDefault) throws JsonParseException, JsonMappingException, IOException {
         ObjectMapper m = new ObjectMapper();
-        return m.readValue(imageResultsAsString, List.class);
+        if (useDefault) {
```

Comment:
This will be cleaner & more extensible if, instead of passing in a boolean, just pass in string to test.  That is,
```
List<Map> getExpectedResults(String expectedResults)
...
return m.readValue(expectedResults, List.class)
```

As soon as we have to test another image or another result, the boolean -- limit of two choices -- will be insufficient.

This can be done in a subsequent PR, but better to think a bit more future-proof.

Reply:
Agreed this is a better model, I'll change it.